packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -174,9 +217,10 @@ export const SubItem = styled.li`
 export const LegalLinksWrapper = styled.span`
   display: flex;
   align-items: center;
-  margin-top: ${sizes(8)};
+  margin-top: ${sizes(6)};
   padding: ${sizes(4)} 0 ${sizes(6)};
-  border-top: 1px solid ${oldColors.gray[300]};
+  border-top: 1px solid ${cVar('colorBorderMutedAlpha')};
+  font: ${cVar('typographyDesktopT100')};
```

Comment:
Let's remove `font` attribute and use `Text` component with `variant="t100"` prop.

Reply:
It's used all over this file though, should I do that everywhere?

Reply:
Yes, let's replace it where we can